config/site_settings.yml

```diff
@@ -1069,6 +1069,9 @@ embedding:
   embed_post_limit:
     default: 100
     hidden: true
+  embed_link_to_top:
+    default: false
+    hidden: true
```

Comment:
I don't have any sites that embed personally - but this seems like something that would be quite useful to certain types of blogs, so hiding it seemed odd. Not one of the devs though, so just take my comment as an "opinion".